### PR TITLE
Hashing Code Added

### PR DESCRIPTION
**Pull request by aritra-tech**
Added the code for Hashing Algorithm.

Issue No #33

### CODE REVIEW

**Comment by kodeflap** [comment]
I will review it😃

File: app/src/main/java/com/dlight/algoguide/dsa/hashing/Hashing.kt [hunk 1]
```diff
@@ -1,4 +1,148 @@
 package com.dlight.algoguide.dsa.hashing
 
-class Hashing {
+class HashMap<K, V>() {
```
Comment (kodeflap) [inline app/src/main/java/com/dlight/algoguide/dsa/hashing/Hashing.kt:3]: Rename the class name to Hashing

**Review by kodeflap — CHANGES_REQUESTED** [review]
This PR request needs a class name change to avoid merge conflicts

File: app/src/main/java/com/dlight/algoguide/dsa/hashing/Hashing.kt (hunk 1 above)
Comment (aritra-tech) [reply app/src/main/java/com/dlight/algoguide/dsa/hashing/Hashing.kt:3]: Ok I will do the changes

**Comment by aritra-tech** [comment]
I have done the changes @kodeflap

**Review by kodeflap — APPROVED** [review]
Thank you for the modification this PR is approved

**Pull request merged**